### PR TITLE
Add ffmpeg-full

### LINKED ISSUE #94: Can't import WAV AudioClip to AAC for WebGL, because of ffmpeg issue
### Bug
When switching to WebGL platform, .wav AudioClip importations (which require conversion to AAC) fail, logging errors like this to the console in the Unity editor:
> Errors during import of AudioClip Assets/[...].wav:
> ffmpeg -i "Temp/fsbankcache 9/tmpinput_ZyLBhb.WAV" -f mp4 -codec aac -b:a 192000 -strict experimental -y "Temp/importedAudioTempPrimary"
> FSBTool ERROR: Error converting Temp/fsbankcache 9/tmpinput_ZyLBhb.WAV: make sure you have avconv or ffmpeg installed
> FSBTool ERROR: Failed encoding audio clip '/home/[...].wav' to AAC. Possibly the file is too short. Try to append silence such that the length becomes at least 1024 samples.

File is not too short and `ffmpeg` is installed. So, when trying to run that command line manually:
```sh
ffmpeg -i "Temp/fsbankcache 9/tmpinput_ZyLBhb.WAV" -f mp4 -codec aac -b:a 192000 -strict experimental -y "Temp/importedAudioTempPrimary"
```
It works fine outside the flatpak. The `importedAudioTempPrimary` file is generated and has AAC format.
<details><summary>Command output outside sandbox:</summary>

```
ffmpeg version 5.1.2-3ubuntu1 Copyright (c) 2000-2022 the FFmpeg developers
  built with gcc 12 (Ubuntu 12.2.0-14ubuntu2)
  configuration: --prefix=/usr --extra-version=3ubuntu1 --toolchain=hardened --libdir=/usr/lib/x86_64-linux-gnu --incdir=/usr/include/x86_64-linux-gnu --arch=amd64 --enable-gpl --disable-stripping --enable-gnutls --enable-ladspa --enable-libaom --enable-libass --enable-libbluray --enable-libbs2b --enable-libcaca --enable-libcdio --enable-libcodec2 --enable-libdav1d --enable-libflite --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libglslang --enable-libgme --enable-libgsm --enable-libjack --enable-libmp3lame --enable-libmysofa --enable-libopenjpeg --enable-libopenmpt --enable-libopus --enable-libpulse --enable-librabbitmq --enable-librist --enable-librubberband --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libsrt --enable-libssh --enable-libsvtav1 --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx265 --enable-libxml2 --enable-libxvid --enable-libzimg --enable-libzmq --enable-libzvbi --enable-lv2 --enable-omx --enable-openal --enable-opencl --enable-opengl --enable-sdl2 --disable-sndio --enable-libjxl --enable-pocketsphinx --enable-librsvg --enable-libmfx --enable-libdc1394 --enable-libdrm --enable-libiec61883 --enable-chromaprint --enable-frei0r --enable-libx264 --enable-libplacebo --enable-librav1e --enable-shared
  libavutil      57. 28.100 / 57. 28.100
  libavcodec     59. 37.100 / 59. 37.100
  libavformat    59. 27.100 / 59. 27.100
  libavdevice    59.  7.100 / 59.  7.100
  libavfilter     8. 44.100 /  8. 44.100
  libswscale      6.  7.100 /  6.  7.100
  libswresample   4.  7.100 /  4.  7.100
  libpostproc    56.  6.100 / 56.  6.100
Guessed Channel Layout for Input Stream #0.0 : mono
Input #0, wav, from 'Temp/fsbankcache 9/tmpinput_ZyLBhb.WAV':
  Duration: 00:00:57.04, bitrate: 705 kb/s
  Stream #0:0: Audio: pcm_s16le ([1][0][0][0] / 0x0001), 44100 Hz, mono, s16, 705 kb/s
Stream mapping:
  Stream #0:0 -> #0:0 (pcm_s16le (native) -> aac (native))
Press [q] to stop, [?] for help
Output #0, mp4, to './importedAudioTempPrimary':
  Metadata:
    encoder         : Lavf59.27.100
  Stream #0:0: Audio: aac (LC) (mp4a / 0x6134706D), 44100 Hz, mono, fltp, 192 kb/s
    Metadata:
      encoder         : Lavc59.37.100 aac
size=     937kB time=00:00:57.05 bitrate= 134.5kbits/s speed=16.3x    
video:0kB audio:927kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 1.123389%
[aac @ 0x55721c063780] Qavg: 65406.895
```
</details>

But inside `flatpak run --command=sh com.unity.UnityHub`, the same command fails.
<details><summary>Command output inside sandbox:</summary>

```
ffmpeg version 4.3.5 Copyright (c) 2000-2022 the FFmpeg developers
  built with gcc 11.3.0 (GCC)
  configuration: --prefix=/usr --libdir=/usr/lib/x86_64-linux-gnu --disable-debug --disable-doc --disable-static --enable-optimizations --enable-shared --disable-everything --enable-ffplay --enable-ffprobe --enable-gnutls --enable-libaom --enable-libdav1d --enable-libfdk-aac --enable-libmp3lame --enable-libfontconfig --enable-libfreetype --enable-libopus --enable-libpulse --enable-libspeex --enable-libtheora --enable-libvorbis --enable-libvpx --enable-libwebp --enable-openal --enable-opengl --enable-sdl2 --enable-vulkan --enable-zlib --enable-libv4l2 --enable-libxcb --enable-vdpau --enable-vaapi --enable-encoder='ac3,alac,flac,libfdk_aac,g723_1,mp2,libmp3lame,libopus,libspeex,pcm_alaw,pcm_mulaw,pcm_f32le,pcm_s16be,pcm_s24be,pcm_s16le,pcm_s24le,pcm_s32le,pcm_u8,tta,libvorbis,wavpack,' --enable-encoder='ass,ffv1,libaom_av1,libvpx_vp8,libvpx_vp9,mjpeg_vaapi,rawvideo,theora,vp8_vaapi,libopenh264' --enable-decoder='adpcm_g722,alac,flac,g723_1,g729,libfdk_aac,libopus,libspeex,mp2,mp3,m4a,pcm_alaw,pcm_mulaw,pcm_f16le,pcm_f24le,pcm_f32be,pcm_f32le,pcm_f64be,pcm_f64le,pcm_s16be,pcm_s16be_planar,pcm_s24be,pcm_s16le,pcm_s16le_planar,pcm_s24le,pcm_s24le_planar,pcm_s32le,pcm_s32le_planar,pcm_s64be,pcm_s64le,pcm_s8,pcm_s8_planar,pcm_u8,pcm_u24be,pcm_u24le,pcm_u32be,pcm_u32le,tta,vorbis,wavpack,' --enable-decoder='ass,ffv1,mjpeg,mjpegb,libaom_av1,libdav1d,libvpx_vp8,libvpx_vp9,rawvideo,theora,vp8,vp9,libopenh264' --enable-encoder='bmp,gif,jpegls,png,tiff,webp,' --enable-decoder='bmp,gif,jpegls,png,tiff,webp,' --enable-hwaccel='vp8_vaapi,mjpeg_vaapi,' --enable-parser='aac,ac3,flac,mjpeg,mpegaudio,mpeg4video,opus,vp3,vp8,vp9,vorbis,' --enable-muxer='ac3,ass,flac,g722,gif,matroska,mp3,mpegvideo,rtp,ogg,opus,pcm_s16be,pcm_s16le,wav,webm,' --enable-demuxer='aac,ac3,ass,flac,g722,gif,image_jpeg_pipe,image_png_pipe,image_webp_pipe,matroska,mjpeg,mov,mp3,mpegvideo,ogg,pcm_mulaw,pcm_alaw,pcm_s16be,pcm_s16le,rtp,wav,' --enable-filter='crop,scale,overlay,amix,amerge,aresample,format,aformat,fps,transpose,pad,' --enable-protocol='crypto,file,pipe,rtp,srtp,rtsp,tcp,udp,unix,' --arch=x86_64 --enable-libopenh264
  libavutil      56. 51.100 / 56. 51.100
  libavcodec     58. 91.100 / 58. 91.100
  libavformat    58. 45.100 / 58. 45.100
  libavdevice    58. 10.100 / 58. 10.100
  libavfilter     7. 85.100 /  7. 85.100
  libswscale      5.  7.100 /  5.  7.100
  libswresample   3.  7.100 /  3.  7.100
Guessed Channel Layout for Input Stream #0.0 : mono
Input #0, wav, from 'Temp/fsbankcache 9/tmpinput_ZyLBhb.WAV':
  Duration: 00:00:57.04, bitrate: 705 kb/s
    Stream #0:0: Audio: pcm_s16le ([1][0][0][0] / 0x0001), 44100 Hz, mono, s16, 705 kb/s
[NULL @ 0x55a7c4c26900] Requested output format 'mp4' is not a suitable output format
Temp/importedAudioTempPrimary: Invalid argument
```
</details>

Note the error part that appears in red:
```
[NULL @ 0x55a7c4c26900] Requested output format 'mp4' is not a suitable output format
Temp/importedAudioTempPrimary: Invalid argument
```
What could be causing this error? Can it be just the version difference in ffmpeg?
`ffmpeg version 5.1.2-3ubuntu1`
`ffmpeg version 4.3.5`
Or is it something else? Is there some workaround to make it use the system's ffmpeg?

### Versions
Unity 2021.3.26f1
Flatpak 1.14.4
UnityHub 3.4.1
Kubuntu 23.04
KDE Plasma 5.27.4
KDE Frameworks 5.104.0
Qt 5.15.8
Linux 6.2.0-20-generic (64-bit)
X11

### PR DESCRIPTION
**Pull request by DekosAnjo**
i fixed the ffmpeg error #94 adding the full ffmpeg module. now Wav files imported correctly on the Webgl

Fixes #94

### CODE REVIEW

**Comment by DekosAnjo** [comment]
@wjt @cassidyjames can you merge this?

**Comment by wjt** [comment]
bot, build

File: com.unity.UnityHub.yaml [hunk 1]
```diff
@@ -30,6 +30,14 @@ add-extensions:
...
     directory: lib/debug/lib/i386-linux-gnu
     version: '22.08'
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '22.08'
+    add-ld-path: .
+  org.freedesktop.Platform.ffmpeg_full.i386:
+    directory: lib32/ffmpeg
+    version: '22.08'
+    add-ld-path: .
```
Comment (wjt) [inline com.unity.UnityHub.yaml:40]: I don't think we need the 32-bit version. From #94, Unity invokes whatever `ffmpeg` is in the path, and the one in the path is the 64-bit version at `/usr/bin/ffmpeg`.

Plus, the given `directory`, `lib32/ffmpeg`, does not exist in `/app`. So at best, defining this extension has no effect; and at worst, if the extension is installed the app may fail to start. (I don't know what Flatpak does if the target directory for an extension is missing.)

```suggestion
```

**Review by wjt — CHANGES_REQUESTED** [review]

**Review by wjt — APPROVED** [review]
Thanks!

**Pull request merged**